Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "intel-pstate"
-version = "0.2.0"
```

Comment:
that's weird. this is version 0.2.1 on master. why is this pull request showing 0.2.0?